src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
Is this needed?

Reply:
It's generated, I guess it probably has some use...